docs/reference/settings/transform-settings.asciidoc

```diff
@@ -9,17 +9,28 @@
 
 You do not need to configure any settings to use {transforms}. It is enabled by default.
 
-All of these settings can be added to the `elasticsearch.yml` configuration file. 
-The dynamic settings can also be updated across a cluster with the 
+All of these settings can be added to the `elasticsearch.yml` configuration file.
+The dynamic settings can also be updated across a cluster with the
 <<cluster-update-settings,cluster update settings API>>.
 
-TIP: Dynamic settings take precedence over settings in the `elasticsearch.yml` 
+TIP: Dynamic settings take precedence over settings in the `elasticsearch.yml`
 file.
 
 [float]
-[[general-data-frames-settings]]
+[[general-transform-settings]]
 ==== General {transforms} settings
 
+`node.transform`::
+Set to `true` (default) to identify the node as a _transform node_. +
```

Comment:
```suggestion
Set to `true` to identify the node as a _transform node_. The default is `false` if either `node.data` or `xpack.transform.enabled` is `false` for the node, and `true` otherwise. +
```

(This assumes the idea from my other comment in this batch is implemented.)